app/population/controllers/edit.js

```diff
@@ -4,6 +4,11 @@ const FormWizardController = require('../../../common/controllers/form-wizard')
 const populationService = require('../../../common/services/population')
 
 class DetailsController extends FormWizardController {
+  middlewareLocals() {
+    super.middlewareLocals()
+    this.use(this.setButtonText)
```

Comment:
Maybe we should think about adding this in to `FormWizardController` now

Reply:
tbh if we were all around on the project more, I'd suggest getting this into the HMPO Form Wizard framework itself